### DISCUSSION TITLE
System dialogs and offers does not respect system localization

### DISCUSSION THREAD

**Discussion by zvitek** [ionic-team/capacitor#4659] [Q&A] [answered]
Hello, 
I have problem with localization of dialogs and offers  in my application. I'm talking about  file upload, calendar, copy/past etc.
My application is in Czech language and system language is Czech too.
But if I want to pick a date or work with text so I have all dialogs in english.
Is there way how to display dialogs in phone localization?

Thank you

![image](https://user-images.githubusercontent.com/6132296/120341209-fac6ad80-c2f6-11eb-95b8-7b01040f41a7.png)
![image](https://user-images.githubusercontent.com/6132296/120341264-05814280-c2f7-11eb-829e-51227d291a83.png)
![image](https://user-images.githubusercontent.com/6132296/120341309-0e721400-c2f7-11eb-8e93-4000f7bf5e37.png)

**Comment by jcesarmobile** [comment, marked as answer]
Check my answer on a similar topic

https://github.com/ionic-team/capacitor/discussions/4263#discussioncomment-419278

**Reply by zvitek** [reply]
Yep, that's it. Thank you
